### PR TITLE
updated psp-validation bglibpy dependency to 4.0.27

### PR DESCRIPTION
**Pull request by mgeplf**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-bglibpy/package.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ class PyBglibpy(PythonPackage):
     git      = "ssh://bbpcode.epfl.ch/sim/BGLibPy"
 
     version('develop', branch='master')
-    version('4.0.17', commit='e90513f52a7d9ca3c16877ae7a2bcb8df31c8545', preferred=True)
+    version('4.0.28', commit='9205075489845fc389b2f2f8e9637dfa14345ee6', preferred=True)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-bglibpy/package.py]: Could we use tags, please?

**Review by matz-e — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-bglibpy/package.py (hunk 1 above)
Comment (mgeplf) [reply var/spack/repos/builtin/packages/py-bglibpy/package.py]: I'm not in charge of the bglibpy project, and it only has tags for major versions.  It uses versioneer, so I don't know how that works with tags.

Comment (mgeplf) [reply var/spack/repos/builtin/packages/py-bglibpy/package.py]: Maybe @wvangeit can comment

Comment (wvangeit) [reply var/spack/repos/builtin/packages/py-bglibpy/package.py]: See here https://github.com/BlueBrain/spack/pull/419

**Comment by pramodk** [comment]
@mgeplf : #419 is merged, please rebase.

**Comment by mgeplf** [comment]
I wish I had know that there was another commit in flight that would have fixed my problem.  Does github not have something similar to gerrit's 'Conflicts With' warning?

File: var/spack/repos/builtin/packages/py-bglibpy/package.py (hunk 1 above)
Comment (matz-e) [reply var/spack/repos/builtin/packages/py-bglibpy/package.py]: Also, please don't remove old versions while they are still being referenced as dependencies. There's no harm in leaving a (maybe short-ish) list of versions.

Comment (mgeplf) [reply var/spack/repos/builtin/packages/py-bglibpy/package.py]: With python3, the `4.0.17` wouldn't work, so I don't think it's worth leaving around.

**Review by wizmer — APPROVED** [review]

File: var/spack/repos/builtin/packages/py-bglibpy/package.py (hunk 1 above)
Comment (wvangeit) [reply var/spack/repos/builtin/packages/py-bglibpy/package.py]: I agree with @mgeplf

**Review by matz-e — APPROVED** [review]
Just a dependency change, LGTM.

**Comment by mgeplf** [comment]
Do I have to trigger a retest or something here?

**Comment by matz-e** [comment]
Retest this please.

Apparently so. The ways of Jenkins are mythical.

**Comment by mgeplf** [comment]
Thanks

**Pull request merged**